src/document/UBSortFilterProxyModel.h

```diff
@@ -10,6 +10,6 @@ class UBSortFilterProxyModel : public QSortFilterProxyModel
     UBSortFilterProxyModel();
 
     bool lessThan(const QModelIndex &left, const QModelIndex &right) const;
+    bool filterAcceptsRow(int sourceRow, const QModelIndex &sourceParent) const;
```

Comment:
In the base class, `filterAcceptsRow()` is protected. A derived class should not change visibility of the function. I therefore propose to add `protected:`. And as it is the same with `lessThan()`, I would add this before line 12 and fix it for this function, too.

And as both functions are overriding a base class function, I would also add `override` to both.

And an unrelated side remark: This header contains a completely unnecessary `#include "core/UBPersistenceManager.h"`. I know, this has nothing to do with this PR, but as we'er already touching this header, shouldn`t we also clean up that? Typically I do not add unrelated cleanups to a PR, but here it is so obvious...

Reply:
Ok, it is done.
Me too, I am not comfortable to change code with no relation to the PR. But it is done.
When I remove `#include "core/UBPersistenceManager.h"`, it leads me to adjust other includes on unrelated files.

Reply:
> When I remove `#include "core/UBPersistenceManager.h"`, it leads me to adjust other includes on unrelated files.

But on each of those occurrences you fixed a "bad" include. Those files did only compile because they were relying on the side effect from the `UBSortFilterProxyModel.h`.